Open-Rival/src/UnitAnimation.cpp

```diff
@@ -45,7 +50,14 @@ namespace Rival {
     }
 
     void UnitAnimation::tick() {
-        auto spritesheetSpan = spritesheetEntry.second - spritesheetEntry.first + 1;
-        animationStep = (animationStep + 1) % (spritesheetSpan);
+        auto spritesheetSpan = std::get<1>(spritesheetEntry) - std::get<0>(spritesheetEntry) + 1;
+        if (1 == spritesheetSpan) {
```

Comment:
I've noticed that you tend to write your conditionals "backwards". That is, I would instinctively write, `if (spritesheetSpan == 1)`. Just curious, what's the reasoning here?

Reply:
In case you forgot or simply your keyboard did not register a single "=", `spritesheetSpan = 1` would compile but might go unnoticed. If you write `1 = spritesheetSpan`, then your code wouldn't compile. A habit which saves trouble occasionally.

Reply:
I see what you mean, although I'm not sure it's worth the impact on readability.

Reply:
Changed, reluctantly :)